Riot/Modules/Application/LegacyAppDelegate.m

```diff
@@ -394,6 +394,12 @@ - (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:
     [NSBundle mxk_setLanguage:language];
     [NSBundle mxk_setFallbackLanguage:@"en"];
     
+    if (BuildSettings.disableRightToLeftLayout)
+    {
+        [[UIView appearance] setSemanticContentAttribute:UISemanticContentAttributeForceLeftToRight];
+        [[UIView appearanceWhenContainedInInstancesOfClasses:@[UIAlertController.class]] setSemanticContentAttribute:UISemanticContentAttributeUnspecified];
```

Comment:
I'd maybe leave a comment explaining why we force all views to LTR but leave alerts unspecified

Reply:
Ah thanks you're right. This allowed the buttons to be ordered RTL if the were side-by-side, but given I added an explicit BuildSetting to disable RTL, it doesn't actually make any sense now. Removed 👍